src/common/CryptoUtils.ts

```diff
@@ -132,6 +132,72 @@ export class CryptoUtils {
         };
     }
 
+    /**
+     * Converts a certificate from PEM to DER format.
+     *
+     * @param pemCertificate A string representing a certificate in PEM format.
+     * @returns A buffer containing the data of the certificate in DER format.
+     */
+    public static PEMtoDER(pemCertificate: string): Buffer {
+        const cert = this.getCertFromPEM(pemCertificate);
+        const derCertString = forge.asn1.toDer(forge.pki.certificateToAsn1(cert)).getBytes();
+        const derCertBuffer = Buffer.from(derCertString, 'binary');
+        return derCertBuffer;
+    }
+
+    /**
+     * Converts a certificate from DER to PEM format.
+     *
+     * @param derCertificate A buffer containing the binary data of the certificate in DER format.
+     * @returns A string representing a certificate in PEM format.
+     */
+    public static DERtoPEM(derCertificate: Buffer): string {
+        const cert = this.getCertFromDER(derCertificate);
+        const pemCert = forge.pki.certificateToPem(cert);
+        return pemCert;
+    }
+
+    /**
+     * Computes the subject hash (similar to subject_hash_old of OpenSSL). When invoking
```

Comment:
Nit:

```suggestion
     * Computes the old-style (MD5) subject hash (similar to subject_hash_old of OpenSSL) that Android uses to label its user certificates. When invoking
```

I went with some of the description convention that OpenSSL itself [uses](https://github.com/openssl/openssl/blob/e8498dc6455fc36f70dc3a0ca1ef82b34c088a90/apps/x509.c#L108-L111).